torch/onnx/__init__.py

```diff
@@ -130,13 +129,6 @@
 producer_version = _C_onnx.PRODUCER_VERSION
 
 
-@_deprecation.deprecated(
-    since="1.12.0", removed_in="2.0", instructions="use `torch.onnx.export` instead"
```

Comment:
Do we really want to delete "use `torch.onnx.export` instead" message?

Reply:
I think so? Because it’s been there for longer than the depreciation period

Reply:
I mean now people will use `torch.onnx.utils._export` directly and can't know that they are supposed to use `torch.onnx.export`.

IMHO the message should be moved to the docstring of `torch.onnx.utils._export` instead of deleting.

Reply:
It’s a private function though. Users should expect no guarantees when they use it in the first place?

Reply:
`torch.onnx._export` was also technically private. Needs some info somewhere to nudge people to use `torch.onnx.export`.

Reply:
`torch.onnx._export` is a weird thing. not fully private nor public. The fact that it started with `_` made it private by convention, but the fact that it was public and explicitly exposed through `torch.onnx.__init__.py` made it officially public too.

`torch.onnx._export` warned users for a couple releases about its deprecation. So it should be ok to delete both the message and public API for it. Nonetheless, a comment at `torch.onnx.utils._export` is a nice to have touch, though. I don't see why not just add a note there for future references.